environments/production/postgresql.yml

```diff
@@ -72,43 +72,43 @@ dbs:
   form_processing:
     proxy:
       host: pgbouncer3
-      pgbouncer_endpoint: pgbouncer3
+      pgbouncer_endpoint: pgshard_nlb
```

Comment:
I don't think we want to do this part yet, because only pgbouncer3 is capable of serving the pl/proxy database, which then routes to the shard databases. That was my understanding, did anything change there or was this just an oversight? I have not been following very closely.